src/__tests__/api.test.ts

```
@@ -0,0 +1,85 @@
+                updatedAt: new Date(),
+            });
+            const request = new Request('http://localhost:3000/api/users', {
+                method: 'POST',
+                body: JSON.stringify({
+                    email: 'new@example.com',
+                    name: 'New User'
+                }),
+                headers: { 'Content-Type': 'application/json' }
+            });
+
+            const response = await POST(request as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)